src/main/java/br/com/tech/challenge/utils/Utils.java

```diff
@@ -0,0 +1,16 @@
+package br.com.tech.challenge.utils;
+
+import br.com.tech.challenge.api.exception.StatusPedidoInvalidoException;
+import br.com.tech.challenge.domain.enums.StatusPedido;
+
+public class Utils {
+
+    public static void validarStatusPedido(StatusPedido statusPedido) {
```

Comment:
Faz sentido essa checagem estar aqui? Por ser do domínio de pedidos, acredito que esse método poderia estar dentro do service correspondente. A não ser que a ideia era aproveitar esse método em outros locais fora do escopo, mas ão sei se é o caso. Se for, acredito que pdoeria ser criada uma classe com essa validação com um nome menos genérico do que só "Utils"

Reply:
tinha deixado realmente no service mas como voltei uma versão pra atualizar com o código de vcs acabei subindo um utils que uso no meu ambiente de dev local.